cmd/controller/app/controller.go

```diff
@@ -80,7 +81,7 @@ func Run(opts *config.ControllerConfiguration, stopCh <-chan struct{}) error {
 	}
 
 	enabledControllers := options.EnabledControllers(opts)
-	log.Info(fmt.Sprintf("enabled controllers: %s", enabledControllers.List()))
+	log.Info(fmt.Sprintf("enabled controllers: %s", sets.List(enabledControllers)))
```

Comment:
`sets.String.List` returned an ordered list so I've replaced that with the `sets.List` everywhere, in case the ordering is significant and for backwards compatibility.